weatherpro/weatherpro.py

```
@@ -0,0 +1,1337 @@
+                        await self.config.glossary_definitions_shown.set(glossary_definitions_shown + 1)
+                elif str(reaction) == "❌":
+                    await message.delete()
+                    break
+
+    @commands.guild_only()
+    @weather.command(name="alerts")
+    async def alerts(self, ctx):
+        """Shows a statistical summary of active weather alerts"""
+        url = "https://api.weather.gov/alerts/active/count"
+        async with aiohttp.ClientSession() as session:
+            async with session.get(url) as response:
```

Ruff (0.8.2)

Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)